findCountours() outputs are changed at OpenCV Ver4.1

This error occurred when running chest_camera_example or head_camera_example.

```bash
[ERROR] [1556079683.237687]: bad callback: <bound method ObjectTracker._image_callback of <__main__.ObjectTracker instance at 0x7ff7621bbe60>>
Traceback (most recent call last):
  File "/opt/ros/melodic/lib/python2.7/dist-packages/rospy/topics.py", line 750, in _invoke_callback
    cb(msg)
  File "/home/shotaak-rt/catkin_ws/src/sciurus17_ros/sciurus17_examples/scripts/chest_camera_tracking.py", line 45, in _image_callback
    output_image = self._detect_orange_object(input_image)
  File "/home/shotaak-rt/catkin_ws/src/sciurus17_ros/sciurus17_examples/scripts/chest_camera_tracking.py", line 125, in _detect_orange_object
    return self._detect_color_object(bgr_image, lower_orange, upper_orange)
  File "/home/shotaak-rt/catkin_ws/src/sciurus17_ros/sciurus17_examples/scripts/chest_camera_tracking.py", line 90, in _detect_color_object
    _, contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
ValueError: need more than 2 values to unpack

```

I fixed python scripts and the examples worked correctly.

```diff
        # マスクから輪郭を抽出
-       _, contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+        contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
```

--- a/sciurus17_examples/scripts/chest_camera_tracking.py
+++ b/sciurus17_examples/scripts/chest_camera_tracking.py
@@ -30,6 +30,8 @@ class ObjectTracker:
         self._image_shape = Point()
         self._object_detected = False
 
+        self._CV_MAJOR_VERSION, _, _ = cv2.__version__.split('.')
+
 
     def _image_callback(self, ros_image):
         try:
@@ -87,7 +89,10 @@ class ObjectTracker:
         mask = cv2.inRange(hsv, lower_color, upper_color)
 
         # マスクから輪郭を抽出
-        _, contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+        if self._CV_MAJOR_VERSION == '4':
+            contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+        else:
+            _, contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
 
         # 輪郭を長方形に変換し、配列に格納
         rects = []

--- a/sciurus17_examples/scripts/depth_camera_tracking.py
+++ b/sciurus17_examples/scripts/depth_camera_tracking.py
@@ -42,6 +42,8 @@ class ObjectTracker:
         self._image_shape = Point()
         self._object_detected = False
 
+        self._CV_MAJOR_VERSION, _, _ = cv2.__version__.split('.')
+
 
     def _image_callback(self, ros_image):
         try:
@@ -130,7 +132,10 @@ class ObjectTracker:
             mask = cv2.inRange(input_depth_image, depth[0], depth[1])
 
             # マスクから輪郭を抽出
-            _, contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+            if self._CV_MAJOR_VERSION == '4':
+                contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+            else:
+                _, contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
 
             # 輪郭を長方形に変換し、配列に格納
             rects = []

--- a/sciurus17_examples/scripts/head_camera_tracking.py
+++ b/sciurus17_examples/scripts/head_camera_tracking.py
@@ -30,6 +30,8 @@ class ObjectTracker:
         self._image_shape = Point()
         self._object_detected = False
 
+        self._CV_MAJOR_VERSION, _, _ = cv2.__version__.split('.')
+
         # カスケードファイルの読み込み
         # 例
         # self._face_cascade = cv2.CascadeClassifier("/home/USER_NAME/.local/lib/python2.7/site-packages/cv2/data/haarcascade_frontalface_alt2.xml")
@@ -95,7 +97,10 @@ class ObjectTracker:
         mask = cv2.inRange(hsv, lower_color, upper_color)
 
         # マスクから輪郭を抽出
-        _, contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+        if self._CV_MAJOR_VERSION == '4':
+            contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
+        else:
+            _, contours, _ = cv2.findContours(mask, cv2.RETR_TREE, cv2.CHAIN_APPROX_SIMPLE)
 
         # 輪郭を長方形に変換し、配列に格納
         rects = []

